driver/venstar/driver.go

```diff
@@ -53,6 +77,75 @@ func (d *Driver) Start() {
 	}()
 	d.iface = d.svc.RegisterInterface(d.r.Name, "i.venstar")
 	d.iface.SubscribeSlot("control", d.Control)
+
+	d.xbos_iface = d.svc.RegisterInterface(d.r.Name, "i.xbos.thermostat")
+	d.xbos_iface.SubscribeSlot("setpoints", func(msg *bw2.SimpleMessage) {
+		fmt.Println("got message from slot setpoints:")
+		msg.Dump()
+		cm := make(map[string]interface{})
```

Comment:
Oh whoops. That one doesn't seem to compile on OSX because of "undefined: syscall.SO_BINDTODEVICE" so I didn't see this issue.